packages/webcomponents/src/components/form/form-helpers/form-help-text/form-help-text.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'form-help-text',
+})
+export class FormHelpText {
+  @Prop() helpText: string;
+  @Prop() errorText: string;
+
+  private text: string;
+
+  get colorClass() {
+    return this.errorText ? 'text-danger' : 'text-muted';
+  }
+
+  render() {
+
+    if (!this.helpText && !this.errorText) {
+      return null;
+    }
+
+    if (this.errorText) {
```

Comment:
Would it be better to keep help text and error text separate? We may want the flexibility of displaying both? I think with the way this is we are coupling the two when the purposes are possibly too different

Reply:
Agreed, and done. Good call. 👍🏻